File: 16 DBA2 Acceso a Mongo con MongoRepository y MongoTemplate/src/main/java/com/mongodbcrud/mongodbcrud/Domain/Person.java
package com.mongodbcrud.mongodbcrud.Domain;

import lombok.Data;
import lombok.Getter;
import lombok.Setter;
import lombok.ToString;
import org.bson.types.ObjectId;
import org.springframework.data.annotation.Id;
import org.springframework.data.mongodb.core.mapping.Document;
import org.springframework.data.mongodb.core.mapping.MongoId;

import java.sql.Timestamp;
import java.time.Instant;
import java.util.Date;

@Document(collection = "person")
@Data
public class Person {
    @MongoId
    @Id
    ObjectId _id;

    String user;

    String password;

    String name;

    String surname;

    String company_email;

    String personal_email;

    String city;

    Boolean active;

    Date created_date = Timestamp.from(Instant.now());

    String image_url;

    Date termination_date;
}

File: 24 NodejS - CRUD Personas Sequelize/src/person/domain/Person.ts
import {
  AutoIncrement,
  Column,
  CreatedAt,
  Model,
  PrimaryKey,
  Table,
} from "sequelize-typescript";

@Table({
  tableName: "persons",
})
export class Person extends Model {
  @PrimaryKey
  @AutoIncrement
  @Column
  declare id: number;

  @Column
  declare user: string;

  @Column
  declare name: string;

  @Column
  declare surname: string;

  @Column
  declare companyEmail: string;

  @Column
  declare personalEmail: string;

  @Column
  declare city: string;

  @Column
  declare isActive: boolean;

  @CreatedAt
  declare createdAt: Date;

  @Column
  declare imageUrl: string;

  @Column
  declare terminationDate: Date;
}
